source/languages/en/riak/dev/taste-of-riak/nodejs.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Taste of Riak: NodeJS"
+project: riak
+version: 1.4.0+
+document: guide
+toc: true
+audience: beginner
+keywords: [developers, client, javascript, nodejs]
+---
+
+If you haven't set up a Riak Node and started it, please visit the [[Prerequisites|Taste of Riak: Prerequisites]] first.
+
+To try this flavor of Riak, a working installation of NodeJS 0.12 or later is required.
+
+Code for these examples is available [here](https://github.com/basho/riak-nodejs-client-examples/blob/master/dev/taste-of-riak/introduction.js). To run, follow these directions:
```

Comment:
These examples will rarely, if ever, change. Linking to `master` should be fine in this case. If they do change I believe the best scenario is for links from the docs to go to the latest version.

You'll also find line number links later on which I realize are fragile but, in my opinion, much better than copying large amounts of code into the docs themselves. Maintenance will be required either way should ToR ever be changed in the future.